model/src/test/kotlin/PackageCurationTest.kt

```diff
@@ -409,6 +410,23 @@ class PackageCurationTest : WordSpec({
     }
 
     "isApplicable()" should {
+        "accept an empty name and / or version" {
+            val curation = PackageCuration(
+                id = Identifier("Maven:com.android.tools"),
+                data = PackageCurationData(
+                    authors = sortedSetOf("The Android Open Source Project"),
```

Comment:
Setting authors and concluded license is not needed and should be dropped.

Reply:
Actually, I added this deliberately even if not strictly needed to showcase the idea described in the [comment linked from the commit message](https://github.com/oss-review-toolkit/ort/issues/5105#issuecomment-1108228824). I'll also add a code comment.

Reply:
The test is about the matching, not about overriding the `authors` and `concluded license` property. 
I doubt the reader of the test will be aware of the comment link you provided above.
So, from the perspective of that reader the data would be irrelevant as well I guess.